### ISSUE TITLE
SyntaxError 

### ISSUE THREAD

**Issue by xiucz** [yfukasawa/LongQC#2]
hi, @yfukasawa 
When I tried to run longQC, it gave we following error: 
```
python ~/LongQC/longQC.py -h
  File "~/longQC.py", line 255
    le.exec(*le_args, out=cov_path, err=cov_path_e)
          ^
SyntaxError: invalid syntax
```
It seems thant "exec" is a key word in python, how to avoid this?

Thank you,
Xiucz.

**Comment by PerisD** [comment]
Hi @yfukasawa nice addition to the Quality assesment of reads. I ran longQC.py and I get the same error than @xiucz. In addition when I search "path_minimap2 = /path/to/minimap2_mod/" in lonqQC.py to modify the PATH, the line code is not the same as the one described in the github, instead the next "path_minimap2  = os.path.join(os.path.dirname(os.path.abspath(__file__)), "minimap2_mod")". 
Thanks,
Peris

**Comment by yfukasawa** [comment]
Hi @xiucz and @PerisD

Thank you for contacting, and sorry to hear some error happened.
First of all, can I ask python version you used? You can get the version by typing below.

`python --version`

And thank you for pointing an issue in ReadMe, Peris.
I'll modify that part. You don't need to change that part in the current version.

Yoshinori

**Comment by PerisD** [comment]
Thank you for your work!

I am using python 3.6 installed in conda environment. All conda modules are installed through conda.
Does it help?
Peris

**Comment by yfukasawa** [comment]
Hi @PerisD,

Thank you for inputs, and it does help.
I wondered it could be caused by older Python version like 2.7. 
Tentatively, I made a clean environment for python 2, and this issue was reproduced.
You use python 3.6, don't you? Do you use python installed by conda?

```
user@2625fce49897:~$ python --version
Python 2.7.16 :: Anaconda, Inc.
user@2625fce49897:~$ python ~/LongQC/longQC.py
  File "/home/user/LongQC/longQC.py", line 254
    le.exec(*le_args, out=cov_path, err=cov_path_e)
          ^
SyntaxError: invalid syntax
```

@xiucz
if you use python 2, LongQC doesn't work at present. Could you try with python3?
 
Best,
Yoshinori

**Comment by PerisD** [comment]
Hi @yfukasawa,
I figure it out what was going on with python. Now, after installing a module call h5py it worked.
The problem was that I first load anaconda2, later I defined the PYTHONPATH to my python-packages in bashrc and I load the conda environment with python3. But the python version loaded was python2 from the anaconda2 (surprisingly). When I forced to use the python3 (using the complete path to the python bin), the packages path was pointing to the anaconda3 installation where I was missing some of the packages required by LongQC. So I started to be confused. To fix the problem I first load anaconda3, I define the PYTHONPATH for python3 packages and I load the conda environment with python3. And calling the longqc.py --help shows the help information.
Thanks again, I will take a look how the program performs once I have a chance.
Cheers,
Peris
